nectar/downloaders/threaded.py

```diff
@@ -319,7 +319,7 @@ def _fetch(self, request):
             self.fire_download_progress(report)
 
         except SkipLocation:
-            _logger.debug("Skipping {url} because {netloc} could not be reached.".format(
+            _logger.info("Skipping {url} because {netloc} could not be reached.".format(
```

Comment:
It looks like [we are logging the initial failure and that subsequent calls to `netloc` are going to be skipped](https://github.com/daviddavis/nectar/blob/75cbd2517e283a87cdb4504590f1d23860430335/nectar/downloaders/threaded.py#L328-L329) so just leave this as `debug` then?